src/genophenocorr/analysis/predicate/genotype/_geno_bool_rec.py

```diff
@@ -321,3 +322,62 @@ def __str__(self):
 
     def __repr__(self):
         return f'ProtFeaturePredicate(tx_id={self._tx_id}, exon_number={self._pf_name})'
+
+class RecessiveProtRegionPredicate(RecessiveGroupingPredicate):
+    """
+    `ProtRegionPredicate` tests if the `patient` has a variant that overlaps with a given region of the protein.
+
+    The predicate needs the start and end coordinate for the protein region, given as a `Region`.
+    For instance, Region(150, 175)
+
+    :param transcript_id: the accession of the transcript of interest.
+    :param protein_region: a `Region` with the start and end coordinates.
+    """
+
+    def __init__(self, transcript_id: str, protein_region: Region) -> None:
+        self._tx_id = transcript_id
+        self._prot_region = hpotk.util.validate_instance(protein_region, Region, 'protein_region_1')
+
+    def get_question(self) -> str:
+        return f'Variant that affects an amino acid between {self._prot_region.start} and {self._prot_region.end} on ' \
+            f'protein encoded by transcript {self._tx_id}'
+
+    def test(self, patient: Patient) -> typing.Optional[Categorization]:
+        """A protein_region and ProteinMetadataService is given when initializing the class.
```

Comment:
Inaccurate docstring.

- there is no protein metadata service
- we should not be talking about variant effects here, we merely look at the variant location. Therefore, it would be better to just mention testing for presence of variants that overlap with the protein region of interest.